duckduckgo/iOS
# Integrates the latest BSK changes

Task/Issue URL: https://app.asana.com/0/0/1205518995339760/f

BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/505
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1631

## Description

Integrates the latest BSK changes.

## Steps to test this PR

1. Open Console.app and select your phone.  Start the console.
2. Filter by "is disabled"
3. Start NetP and make sure you see the message "The connection tester is disabled"

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
